grid/recruitment/list-candidates/maps/__snapshots__/greenhouse.test.ts.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`recruitment/list-candidates/greenhouse ListCandidates when searching for candidates with invalid job ID returns not found error 1`] = `
+Err {
+  "error": "JessieError: Error in Jessie script
```

Comment:
This suggests that the map (correctly) errors out when an invalid job ID is passed, but the error that is returned isn't helpful to the user. It errors out not because the map detects that the server has responded with a "not found", but because it crashes because some part of the map expects data where there is none ("Cannot read properties of undefined") Please handle this in the map.